doc/content/getting-started/aws/ecs/updating/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Updating"
+description: ""
+weight: 3
+---
+
+This page describes the steps for updating {{% tts %}} on AWS ECS.
+
+<!--more-->
+
+Most steps involve updating a CloudFormation stack from the templates. Some steps involve some manual work.
+
+## Overview
+
+From AWS' point of view, updating {{% tts %}} can be divided into three parts:
+
+1. CloudFormation templates
+
+2. Docker images
+
+3. Migrations
+
+## Updating CloudFormation stacks
+
+Updating CloudFormation stacks can be done via the AWS console, using the "Create changeset" functionality. All stacks need to be updated in their numbering order. For example, a stack using template `1-1-vpc` needs to be updated before `1-2-bastion`, which needs to be updated before `2-1-db-aurora-master`.
+
+Not all stacks require a replacement in each version update. Please refer to `UPGRADING.md`, which contains per-version changes in templates. Only templates mentioned in that file have changed in given version. Moreover use that file as a general guideline what changes to expect.
+
+Make sure deployed templates correspond to your {{% tts %}} version. When updating, pay extra attention to:
+- new parameters
+- removed parameters
+- parameters with changed possible values
+
+as default parameters might not be valid in context of your deployment.
+
+### `2-5-db-timescale`
+
+This template describes an EC2 machine with EBS storage. Currently AWS does not permit replacement updates on EC2 machines with EBS volumes attached. Continuing with update might result in data loss. For this reason examine carefully what changes AWS wants to perform while updating this stack, and potentially apply (some of them) them manually.
+
+As machine image, we use the newest Amazon Linux 2 image. This means that most likely AMI is going to change between two consecutive releases of {{% tts %}}, and upon update AWS will try to replace the EC2 machine. Instead you should update the software on that EC2 machine manually.
+
+### `5-1-ecs-cluster`
+
+In this template we use an autoscaling group. Updates to `LaunchConfiguration` do not cause any effect on already running machines created using said `LaunchConfiguration`. This means that if this template contains changes, for example different `UserData` or an updated AMI image, you should examine changes and apply them manually.
+
+Please remember that both `5-1-ecs-cluster` and `2-5-db-timescale` use bare Amazon Linux 2 images. This should make it easy to decide whether you actually need to update the machines, and if yes, how exactly.
+
+## Docker images
+
+Templates `5-x` create ECS services, and they use Docker images. For this reason when updating `5-x` stacks don't forget to change used Docker images to newer ones. Since `5-x` stacks have images as parameters, don't forget to update them, even if they aren't mentioned in `UPGRADING.md`.
```

Comment:
```suggestion
Templates `5-x` create ECS services, and they use Docker images. For this reason, when updating `5-x` stacks, do not forget to change used Docker images to newer ones. Since `5-x` stacks have images as parameters, do not forget to update them, even if they are not mentioned in `UPGRADING.md`.
```